equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java

```diff
@@ -3,8 +3,7 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import uk.gov.justice.laa.crime.equinity.historicaldata.generated.dto.*;
-import uk.gov.justice.laa.crime.equinity.historicaldata.model.Crm14DetailsModel;
-import uk.gov.justice.laa.crime.equinity.historicaldata.model.Crm14Model;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.*;
```

Comment:
We do not need to import the full models package.  Surely we can reduce it by importing only the ones we need

Reply:
done